packages/tc-schema-validator/tests/enums.js

```diff
@@ -1,7 +1,9 @@
 /* global it, describe, expect */
 const { SDK } = require('@financial-times/tc-schema-sdk');
 
-const enums = new SDK().rawData.getEnums();
+const sdk = new SDK();
+sdk.init();
```

Comment:
What does the `init` function do if you don't pass it options? the tests will pass if we use `await sdk.ready();` instead as well...is it only for creating the updater?

Reply:
that is a good question. It calls `updater.configure()`, which reads local data or primes the updater to start fetching data, as appropriate. That behaviour is a bit weird. Maybe I should move polling initialisation into init too? Or... I'm not sure what's best.

I will document the behaviour a bit better for now, but don't want to make some API changes at the moment as I don't want to make your roll out of rich relationships harder